covid19dp_submission/etc/example_app_config.yml

```diff
@@ -0,0 +1,12 @@
+app:
+  bcftools_binary: /path/to/bcftools
+  nextflow_binary: /path/to/nextflow
+  validator_binary: /path/to/vcf-validator
+  accessioning_jar_file: /path/to/eva-accession-pipeline.jar
+  accessioning_properties_file: /path/to/accessioning.properties
+  accessioning_instance: instance-10
+  clustering_jar_file: /path/to/eva-accession-clustering.jar
+  clustering_properties_file: /path/to/clustering.properties
```

Comment:
It is not a requirement at this point but for the sake of consistency, it would be good if similar config entries where using the same config structure. 
in [submission automation](https://github.com/EBIvariation/eva-submission/blob/abc06a1a102ea1ad50e6308ba898d3dd726be70d/eva_submission/etc/example_submission_config.yaml#L19) we split these into `executable`, `jars` and application specific sections.

Reply:
Sure, it is a good idea to align with that structure...